src/Installer/Beta.htm

```diff
@@ -9,6 +9,9 @@
 		<h4>Release Notes</h4>
 		<ul>
 			<li>DEV_VERSION_NUMBER<ol>
+				<li>Chorus: Fix overflow error when receiving very large project data with resumable option.</li>
```

Comment:
Well, this is for the Beta, so I think it is technically correct. At some point we need to declare that Beta was actually stable. The master branch is currently claiming to be Alpha, but should soon be Beta. We should do it in some sort of graduation ceremony, maybe play "Pomp and Circumstance"